src/ontology/mondo-ingest.Makefile

```diff
@@ -293,7 +293,7 @@ metadata/%-metrics.json: $(COMPONENTSDIR)/%.owl
 j2:
 	pip install j2cli j2cli[yaml]
 
-documentation: j2 $(ALL_DOCS) mapping-progress-report
+documentation: j2 $(ALL_DOCS) unmapped-terms-docs mapped-deprecated-terms-docs slurp-docs
```

Comment:
Updated `documentation` prereqs. Was missing `mapped-deprecated-terms-docs`; added `slurp-docs`; and was unnecessarily rerunning the full mapping progress report; it only needs to create the documentation from the files already generated.

And actually, these docs will have already been generated earlier in `build-mondo-ingest`, because it calls `mapping-progress-report`, which calls `unmapped-terms-docs` as its final step.

Reply:
We should avoid the word slurp as much as we can.

Reply:
Not in this PR though, we can do it in a future PR

Reply:
Oh ok, sounds good. I just did a few quick renames. But yes, there is a lot of code; goal names and file names, with the word "slurp" in it, as well as help text. I just created an issue:
- #268